app/styles/home.js

```diff
@@ -1,9 +1,106 @@
 export default {
-  container: (props) => {
+  container: ({ theme }) => {
     return {
-      fontSize: '12rem',
       display: 'flex',
-      justifyContent: 'center'
+      flexDirection: 'column',
+      alignItems: 'center',
+      alignSelf: 'center',
+      height: '100%',
+      paddingTop: theme.space[3],
+      paddingBottom: theme.space[3],
+      [`@media (max-width: ${theme.breakpoints.tabletWide})`]: {
+        paddingLeft: theme.space[5],
+        paddingRight: theme.space[5]
+      },
+      [`@media (min-width: ${theme.breakpoints.tabletWide}) and (max-width: ${theme.breakpoints.desktop})`]: {
+        paddingLeft: theme.space[6],
+        paddingRight: theme.space[6]
+      },
+      [`@media (min-width: ${theme.breakpoints.desktop}) and (max-width: ${theme.breakpoints.desktopWide})`]: {
+        paddingLeft: theme.space[8],
+        paddingRight: theme.space[8]
+      },
+      [`@media (min-width: ${theme.breakpoints.desktopWide})`]: {
+        paddingLeft: theme.space[10],
+        paddingRight: theme.space[10]
+      },
+      ':before': {
+      [`@media (max-width: ${theme.breakpoints.desktop})`]: {
+        content: '""',
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: '100%',
+        height: '100%',
+        opacity: .6,
+        zIndex: -1,
+        backgroundColor: theme.colors.canvas,
+        backgroundImage: 'url("/images/cobuy-bg-sml-1080.jpg")',
+        backgroundSize: 'cover'
+      },
+      [`@media (min-width: ${theme.breakpoints.desktop})`]: {
+        content: '""',
```

Comment:
`content` is specific to a `::before` element. is this....

oh, i got confused by the indentation, this is all within a `::before`. ⚡️